binder/postBuild

```diff
@@ -0,0 +1,3 @@
+ROMERO_VERSION=v0.4.8
```

Comment:
```suggestion
#!/bin/sh
set -eu

ROMERO_VERSION=v0.4.8
```
This script is failing, but `set -e` isn't set so errors aren't propagated to the outer build
https://travis-ci.org/ome/omero-guide-r/builds/646552735
```
---> Running in eab2a2c5157f
Loading required package: usethis
Building romero.gateway based on branch master from user ome 
[1] "Checking out version v0.4.8"
Warning message:
In install.packages(packageFile, repos = NULL, type = "source",  :
  installation of package ‘./romero.gateway_0.4.8.tar.gz’ had non-zero exit status
Removing intermediate container eab2a2c5157f
 ---> a317d90eebdf
Step 62/64 : COPY /repo2docker-entrypoint /usr/local/bin/repo2docker-entrypoint
 ---> ae56d836435d
```

Reply:
Ah of course, so that's why travis often passes when it should actually fail, because I forgot  `set -eu`. Should check all my other scripts for that. Thanks 👍